bin/oref0-get-profile.js

```diff
@@ -37,19 +56,15 @@ if (!module.parent) {
     var pumpsettings_input = params._.slice(0, 1).pop()
     if ([null, '--help', '-h', 'help'].indexOf(pumpsettings_input) > 0) {
       usage( );
-      process.exit(0)
+      process.exit(0);
     }
     var bgtargets_input = params._.slice(1, 2).pop()
     var isf_input = params._.slice(2, 3).pop()
     var basalprofile_input = params._.slice(3, 4).pop()
     var preferences_input = params._.slice(4, 5).pop()
     var carbratio_input = params._.slice(5, 6).pop()
+    var temptargets_input = params._.slice(6, 7).pop()
```

Comment:
maybe this should go as a switch `--temp-targets`?  In general, for future-compat, all positional args except last are mandatory.